doc/source/tune/doc_code/tune.py

```diff
@@ -0,0 +1,60 @@
+# flake8: noqa
+
+# fmt: off
+# __step1_begin__
+from ray import tune
+import ray
+import os
+
+NUM_MODELS = 100
+
+def train_model(config):
+    score = config["model_id"]
+
+    # Import model libraries, etc...
+    # Load data and train model code here...
+
+    # Return final stats. You can also return intermediate progress
+    # using ray.air.session.report() if needed.
+    # To return your model, you could write it to storage and return its
+    # URI in this dict, or return it as a Tune Checkpoint:
+    # https://docs.ray.io/en/latest/tune/tutorials/tune-checkpoints.html
+    return {"score": score, "other_data": ...}
+# __step1_end__
+
+# __step2_begin__
+# Define trial parameters as a single grid sweep.
+trial_space = {
+    # This is an example parameter. You could replace it with filesystem paths,
+    # model types, or even full nested Python dicts of model configurations, etc.,
+    # that enumerate the set of trials to run.
+    "model_id": tune.grid_search([
```

Comment:
nit: change this to numeric to fully mirror the other example?

Reply:
I actually prefer it to be a string, to emphasize it can be an arbitrary thing and not just numbers.